src/index.ts

```
@@ -546,14 +544,9 @@ export class Bigtable {
    */
   createInstance(
     id: string,
-    optionsOrCallback?: InstanceOptions | CreateInstanceCallback,
```

Let's throw an error now that this is required (and add a test):

```ts
if (!options) {
  throw new Error('A configuration object is required to create an instance.');
}
```

---

Done, thanks!